src/main/scala/com/lightbend/rp/sbtreactiveapp/SbtReactiveAppPlugin.scala

```diff
@@ -93,24 +93,14 @@ object SbtReactiveAppPlugin extends AutoPlugin {
 
   object localImport extends docker.DockerKeys
 
-  override def requires = SbtReactiveAppPluginAll && docker.DockerPlugin && AshScriptPlugin && SbtReactiveAppPluginAll
+  override def requires = SbtReactiveAppPluginAll && docker.DockerPlugin && BashStartScriptPlugin && SbtReactiveAppPluginAll
```

Comment:
`AshScriptPlugin` breaks too many features of `sbt-native-packager`, notably when multiple main classes are detected. Just use `BashStartScriptPlugin`.

Trade-off: It adds a few MB to the final Docker image.